dotcom-rendering/src/web/components/SignInGate/displayRule.ts

```diff
@@ -127,26 +121,44 @@ export const canShowSignInGate = (
 				5,
 			) &&
 			isNPageOrHigherPageView(3) &&
-			isValidContentType(CAPI) &&
-			isValidSection(CAPI) &&
-			isValidTag(CAPI) &&
-			!isPaidContent(CAPI) &&
-			!isPreview(CAPI) &&
+			isValidContentType(contentType) &&
+			isValidSection(sectionName) &&
+			isValidTag(tags) &&
+			// hide the sign in gate on isPaidContent
+			!isPaidContent &&
+			// hide the sign in gate on internal tools preview &&
```

Comment:
Is the trailing `&&` in this comment meant to be there? `</nitpick>`

Reply:
👀 Ha, that's odd. I have no idea how that slipped in!